### DISCUSSION TITLE
Energy dashboard is reporting consumption at 1000x

### DISCUSSION THREAD

**Discussion by blacknell** [BottlecapDave/HomeAssistant-OctopusEnergy#1032] [Q&A] [answered]
My energy dashboard (which previously had an Efergy CT clamp as the grid source) is now using `octopus_energy_electricity_REDACTED_current_total_consumption` with the corresponding unit price.

Grid usage is showing 1000x and I've checked the entity data and it looks perfect so I'm at a loss.

FYI. I've also posted to the discord HA energy forum because it looks to me like nothing to do with this integration.

Any ideas?

**Comment by BottlecapDave** [comment, marked as answer]
Hello. You are potentially using the wrong sensor with the dashboard. As per the [docs](https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/setup/energy_dashboard/) you are wanting the accumulative sensor not the non accumulative one. The non accumulative sensor is a bit of a legacy sensor and provides the delta consumption since the last update (so it goes up and down) which might be causing the issues with the energy dashboard.

**Reply by blacknell** [reply]
Your suggestion worked.

I don't fully understand why because the one I was using only ever increases (and reflects the smart meter total) so I though the energy dashboard works out the intra-day numbers. 

Anyway it's working - thank you. Classic way of me thinking I knew what to use and neglecting to RTFM.

**Reply by BottlecapDave** [reply]
Ah sorry I misread the sensor you were using, but glad the dashboard is now reporting what you expect
